core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -175,7 +176,7 @@ object Defaults {
   val ConsumerGroupMinHeartbeatIntervalMs = 5000
   val ConsumerGroupMaxHeartbeatIntervalMs = 15000
   val ConsumerGroupMaxSize = Int.MaxValue
-  val ConsumerGroupAssignors = ""
+  val ConsumerGroupAssignors = List(classOf[RangeAssignor].getName).asJava
```

Comment:
Did we need to do this because the default couldn't be empty?

Reply:
The KIP says that the default should be `uniform, range`.  We don't have the uniform yet so I have already put the range one here.